src/components/search/SuggestionProductCard/SuggestionProductCard.tsx

```diff
@@ -42,8 +48,8 @@ function SuggestionProductCard({ product, index }: SuggestionProductCardProps) {
           onSearchInputSelection?.(name, href)
         }}
       >
-        <CardContent>
-          <CardImage>
+        <CardContent data-fs-card-content>
+          <CardImage data-fs-card-image>
```

Comment:
What do you think about:
```suggestion
        <CardContent data-fs-suggestion-product-card-content>
          <CardImage data-fs-suggestion-product-card-image>
```

If you agree remember to change in all instances in the CSS file as well.

Reply:
I agree!